lib/hirefire/macro/sqs.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+
+module HireFire
+  module Macro
+    module Sqs
+      extend self
+
+      # Counts the approximate number of jobs in the provided SQS queue.
+      #
+      # @example SQS Macro Usage
+      #   HireFire::Macro::Sqs.queue # all queues
+      #   HireFire::Macro::Sqs.queue("email") # only email queue
+      #   HireFire::Macro::Sqs.queue("audio", "video") # audio and video queues
+      #
+      # @param [Array] queues provide one or more queue names, or none for "all".
+      # @return [Integer] the number of jobs in the queue(s).
+      #
+      def queue(*queues)
+        queues = queues.flatten.map(&:to_s)
+        length = 0
+        client = Aws::SQS::Client.new
+        queue_urls = client.list_queues.queue_urls
```

Comment:
That requires that the user credentials have access to List Queues (mine for example has only access to attributes), this should be added to README docs.